collections/_article/whats-missing-in-godot-for-aaa.md

```diff
@@ -0,0 +1,203 @@
+
+This makes it is an ideal target for optimizing with multithreading. There is [an initial proposal](https://github.com/godotengine/godot-proposals/issues/4962) on threaded processing for scene nodes, which should give complex scenes a very significant performance boost.
+
+### Swarms
+
+Scenes, as mentioned before, are designed complex high level behaviors in the hundreds of instances. Still, sometimes, some games require larger amounts of instances but less complex behaviors instead.
+
+This is needed for some types of game mechanics such as:
+
+* Projectiles (bullet hell as example)
+* Units in some types of strategy games with thousands of entitites roaming across a map.
+* Cars/People in city simulators, where thousands appear all across the city.
+* Sandbox style simulations.
+* Complex custom particles that run on CPU.
+* Flocks, swarms, mobs, debris, etc.
+
+More experienced programmers can use the servers directly or even plug C++ code to do the heavy lifting. ECS is often also proposed as a solution for this. Even Compute (which is fully supported in Godot) can be easily use to solve this pattern.
+
+But for the sake of keeping Godot accessible and easy to use, the idea is to create a [swarm system](https://github.com/godotengine/godot-proposals/issues/2380) that takes care of the rendering/physics/etc in large amounts of those objects and the user only has to fill in the code logic.
+
+### Large team VCS support
+
+Godot text file formats are very friendly to version control. They only write what is needed (no redundant information), keep the ordering of sections and are simple enough to understand changes by just looking at them. Few other technologies work as well in this area.
+
+Despite that, this is far from enough to enable large team collaboration. To enable this, Godot VCS support has to improve in several areas:
+
+* Better integration with the filesystem dock.
+* Better real-time refresh of assets if they were modified externally (and checked out).
+* Support for permissions and file locking: Git does not support this out of the box, but Git-LFS and perforce do. This feature is essential for large teams, avoiding conflicts and keeping files protected from uninteded modifications (as example, an artist modifying code or a scene they don't own by mistake).
+
+Unless the support for this is solid, using Godot in large teams will remain difficult.
+
+### Commercial asset store
+
+While for very large studios this is not an area of interest, medium stized studios still rely on significant amount of assets and pre-made functionality. The Asset Library currently existing in Godot only links to projects hosted on GitHub and is unable to be used for commercial assets.
```

Comment:
```suggestion
While for very large studios this is not an area of interest, medium sized studios still rely on significant amount of assets and pre-made functionality. The Asset Library currently existing in Godot only links to projects hosted on GitHub and is unable to be used for commercial assets.
```

Reply:
`medium-sized` would also be better than `medium sized` here.

Reply:
Edited the original suggestion and committed from it :)